media-sound/airsonic-bin/airsonic-bin-10.6.1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN//-bin}"
+SUBSONIC_HOME="/var/lib/${MY_PN}"
+
+DESCRIPTION="Subsonic is a personal media server, opensource fork of Subsonic"
+HOMEPAGE="https://airsonic.github.io/"
+SRC_URI="https://github.com/airsonic/airsonic/releases/download/v${PV}/airsonic.war"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="ffmpeg lame"
+
+DEPEND=""
+RDEPEND="
+	acct-user/airsonic
+	acct-group/airsonic
+	virtual/jre
+	lame? ( media-sound/lame )
+	ffmpeg? ( virtual/ffmpeg )
+"
```

Comment:
It looks like runtime dependency, but in itself this functionality is necessary for work, and use flag is just handles for user who wants something specific.
I added a "plus" to these use flags in ebuild. Thanks.

Reply:
Right, the issue is that it's a binary ebuild, so I guess DEPEND doesn't exist. I will ask to see what the situation here is.

Reply:
since I updated through --force, the actual lines are here https://github.com/gentoo/gentoo/blob/5e1e7476866de855f8eb8615db7ba522addf9b0d/media-sound/airsonic-bin/airsonic-bin-10.6.1.ebuild#L16